index.js

```diff
@@ -31,11 +31,12 @@ var FeatureService = function (url, options) {
   this.options = options
   this.layer = options.layer || 0
   this.timeOut = 1.5 * 60 * 1000
+  var concurrency = this.url.split('//')[1].match(/^service/) ? 16 : 4
```

Comment:
@dmfenton open to using `url.parse` instead? 

https://nodejs.org/docs/latest/api/url.html#url_url_parse_urlstr_parsequerystring_slashesdenotehost

Reason being regexes can be unreliable, and `require('url')` is standard and bundled with node.